### ISSUE TITLE
Menu issue? Fedora 39 Sericea (and others) do not load...

### ISSUE THREAD

**Issue by indigomirage** [netbootxyz/netboot.xyz#1396]
When booting Fedora 39 Sericea (and most other Fedora spins) an error pops up saying that there is no file or directory.

It seems as though the link to mirrors.kernel.org is broken for a few of the spins.  I'm not sure if the issue is in the menu (I'm on v2.0.76) or at the mirrors.kernel.org site.

**Comment by antonym** [comment]
Yeah, I noticed that Sericea was not under Fedora 39 on mirror.kernel.org org... I'll just need to find a replacement mirror that makes sense. I'm open to suggestions or if anyone knows where to open up a mirrors.kernel.org ticket we can do that too.

**Comment by PrototypeActual** [comment]
@antonym I think this might be the place to report it

https://bugzilla.kernel.org
